packages/compile-solidity/package.json

```diff
@@ -34,7 +35,8 @@
     "original-require": "^1.0.1",
     "require-from-string": "^2.0.2",
     "semver": "^7.3.4",
-    "solc": "0.6.9"
+    "solc": "0.6.9",
```

Comment:
Why is this using such an old version?

Reply:
We're only using it for `solc/wrapper`, so it mostly doesn't matter.  It's an old version because there may have been problems with newer versions on Node 10, which we haven't yet officially dropped support for?  I may be wrong about this and there may not be a problem after all.  Once we drop support for Node 10 we should definitely be able to update.

Reply:
> We're only using it for solc/wrapper, so it mostly doesn't matter.

It matters exactly for that reason, as newer wrapper versions have backwards compatibility functionality, so as long as you use the latest wrapper, it is "guaranteed" to work with every previous version. However it does not offer forwards compatibility, so we can create a breaking release binary which won't work with an old wrapper anymore.

Reply:
In this particular case I think you are fine, as the last such change happened for 0.6.9 in the wrapper, but keep in mind that you may encounter breakage in the future.

Reply:
>  It's an old version because there may have been problems with newer versions on Node 10, which we haven't yet officially dropped support for? I may be wrong about this and there may not be a problem after all. Once we drop support for Node 10 we should definitely be able to update.

Hmm, can you report the issue? Circleci runs with node 8 / 10 / 12 on solc-js, and we do not see any problem.

Reply:
And added CI for node 14/16 just to be sure: https://github.com/ethereum/solc-js/pull/548

Reply:
OK, you might want to open a separate issue for this since this is basically unrelated to the contents of this PR.